spec/rails/rails_spec_helper.rb

```diff
@@ -2,8 +2,11 @@
 require 'spec_helper'
 
 Combustion.path = 'spec/rails/stub_app'
-Combustion.initialize! :action_controller,
-                       :action_view
+Combustion.initialize! :action_controller, :action_view do
+  if Rails.gem_version >= Gem::Version.new("7.1.0")
```

Comment:
I had to put this `if` here because this option is only available on Rails 7.1, but I don't know if this is the best way to check for this

Reply:
That works for me. Searching GitHub for `Rails.gem_version` shows the same style conditionals in other gems.